### ISSUE TITLE
valid-jsdoc has unexplained syntax errors

### ISSUE THREAD

**Issue by sssmall** [eslint/eslint#7191]
**Tell us about your environment**

Mac OS X 10.11.6

* **ESLint Version:** 3.1.1
* **Node Version:** 5.12.0
* **npm Version:** 3.8.6

**What parser (default, Babel-ESLint, etc.) are you using?** Babel-ESLint

**Please show your full configuration:**

```js
{
  "extends": ["airbnb"],
  "env": {
    "es6": true,
    "browser": true,
    "jasmine": true
  },
  "parserOptions": {
    "ecmaVersion": 6
  },
  "parser": "babel-eslint",
  "rules": {
    "valid-jsdoc": "warn",
    "arrow-body-style": ["error", "always"],
    "strict": "off",
    "no-console": "off"
  }
}
```

**What did you do? Please include the actual source code causing the issue.**

This code

```js
'use strict';

/**
 * Adds two numbers
 * @param {number} param1 the first number
 * @param {number} param2 the second number
 * @return {number} the sum of the two numbers
 */
function myFunc(param1, param2) {
  return param1 + param2;
}

myFunc(1, 2);
```
results in a linter warning: `3:1  warning  JSDoc syntax error  valid-jsdoc`

**What did you expect to happen?**

No linter warning

**What actually happened? Please include the actual, raw output from ESLint.**

```
/Users/sssmall/workspace/src/test.js
  3:1  warning  JSDoc syntax error  valid-jsdoc
```

**Comment by platinumazure** [comment]
I'm pretty sure these errors are cascaded up from doctrine. This is definitely a pain point, for sure, but this might be an upstream issue.

**Comment by kaicataldo** [comment]
@sssmall can you confirm this also happens with the default parser? I tried this locally with your example code and am not seeing any linting errors.

**Comment by mysticatea** [comment]
@sssmall I could not reproduce this on my local environment. Could you ensure the reproduce step?

**Comment by vitorbal** [comment]
ping @sssmall any chance you have some updates for us on this issue? Please see the questions above from @kaicataldo @mysticatea. Thanks in advance!

**Comment by sssmall** [comment]
Interesting, I'm no longer able to reproduce this issue. I'll go ahead and close it -- can always reopen it if I encounter it again.
